perception/object_recognition/detection/shape_estimation/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake REQUIRED)
```

Comment:
Not sure if you meant to keep the `ament_cmake` calls if you are converting to `ament_cmake_auto`?

Reply:
Thanks for pointing out.
I've deleted and used ament_auto_find_build_dependencies() instead.